src/data/finset/basic.lean

```diff
@@ -1910,6 +1910,29 @@ theorem map_filter {p : β → Prop} [decidable_pred p] :
   (s.map f).filter p = (s.filter (p ∘ f)).map f :=
 eq_of_veq (map_filter _ _ _)
 
+/-- A helper lemma to produce a default proof for `finset.map_disj_union`. -/
+theorem map_disj_union_aux {f : α ↪ β} {s₁ s₂ : finset α} :
+  (∀ a, a ∈ s₁ → a ∉ s₂) ↔ (∀ a, a ∈ map f s₁ → a ∉ map f s₂) :=
```

Comment:
If you add
```lean
lemma forall_mem_map {f : α ↪ β} {s : finset α} {p : β → Prop} :
  (∀ y ∈ s.map f, p y) ↔ ∀ x ∈ s, p (f x) :=
by simp
```
then this proof becomes
```lean
by simp only [forall_mem_map, mem_map']
```

Otherwise LGTM. Thanks!
bors d+